content/blog/policy-management-policy-packs/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Guardrails for Neo: New Policy Management Experience"
+date: 2025-09-17
+authors:
+    - luke-ward
+    - dan-biwer
+    - insights-team
+meta_desc: "Announcing streamlined user experience for Pulumi Policy and new pre-built policy packs, providing the essential guardrails for AI automation with Pulumi Neo."
+allow_long_title: true
+meta_image: "meta.png"
+tags:
+    - pulumi-service
+    - policy-as-code
+    - crossguard
+    - features
+    - user-experience
+    - pulumi-neo
+    - ai
+---
+
+Yesterday, we introduced [Pulumi Neo](/blog/pulumi-neo/), our new AI platform engineer. As we showed in the [launch video](https://www.youtube.com/watch?v=9GB9M2l1OgY), Neo can automate complex infrastructure tasks, dramatically increasing a platform team's velocity. But with this new power comes a critical need for safety and control. You wouldn't give an AI agent the keys to your production cloud without robust, non-bypassable guardrails.
```

Comment:
We need to tie this back into our overall platform, including platform engineering and IDP developer self-service. I do want us of course to reinforce the news of the week too.